README.md

```diff
@@ -1,2 +1,13 @@
 # JUMP-single-cell
-Single cell analysis of the JUMP Cell Painting consortium data
+Single cell analysis of the JUMP Cell Painting consortium data. Please ensure adequate storage as the sqlite plate data is approximately 1.1 TB large.
```

Comment:
```suggestion
Single cell analysis of the JUMP Cell Painting consortium data.
Please ensure adequate storage as the sqlite plate data is approximately 1.1 TB large.
```

We use one-sentence-per-line in our markdown files (nit-picky, I know!) :)

Reply:
Changed!